src/charts/helpers/loading.js

```diff
@@ -0,0 +1,91 @@
+                            <polygon points="559.92 229 583.92 229 583.92 255 559.92 255"/>
+                            <polygon points="101.738 115 125.738 115 125.738 255 101.738 255"/>
+                            <polygon points="305.374 42 329.374 42 329.374 255 305.374 255"/>
+                            <polygon points="509.011 201 533.011 201 533.011 255 509.011 255"/>
+                            <polygon points="203.556 19 227.556 19 227.556 255 203.556 255"/>
+                            <polygon points="407.192 115 431.192 115 431.192 255 407.192 255"/>
+                        </g>
+                    </g>
+                    <polygon fill="#D2D6DF" fill-rule="nonzero" points="0 295 711 295 711 294 0 294"/>
+                </g>
+                <rect class="chart-filter" fill="url(#lgrad)" x="0" y="0" width="300%" height="100%"></rect>
+            </svg>
+        `,
+
+        donut = `
+            <svg class="loading-state donut-load-state--css-wave" xmlns="http://www.w3.org/2000/svg" width="376" height="331" viewBox="0 0 376 331">
+                ${linearGradient}
+                <g fill="none" fill-rule="evenodd">
+                    <g transform="translate(116 107)">
+                        <circle cx="72" cy="72" r="72" stroke="#EFF2F5" stroke-linecap="round" stroke-width="9.6"/>
+                        <path stroke="#D2D6DF" stroke-width="19.2" d="M126.153559,119.524055 C137.264629,106.845712 144,90.2321371 144,72.0444604 C144,32.2554036 111.764502,0 72,0"/>
+                        <circle cx="72" cy="72" r="67.2" fill="#FFF"/>
+                    </g>
+                </g>
+                <rect class="chart-filter" fill="url(#lgrad)" x="0" y="0" width="300%" height="100%"></rect>
+            </svg>
+        `,
+
+        line = `
+            <svg 
+                class="loading-state line-load-state--css-wave" 
+                xmlns="http://www.w3.org/2000/svg" 
+                width="376" height="331"
+                viewBox="0 0 711 325"
+            >
+                ${linearGradient}
+                <path 
+                    id="chart-bg" 
+                    class="chart-bg" 
+                    style="stroke:#C3C6CF;" 
+                    d="M3.4,216.5h707.3 M3.4,160.5h707.3 M3.3,103.5h707.3 M3.3,48.5h707.6 M0.4,276.6H710H0.4z"
+                />
+               <polyline 
+                    id="chart-line" 
+                    class="chart-line" 
+                    style="stroke:#C3C6CF;stroke-width:4;fill:none;" 
+                    points="8.8,175.8 62.4,237.7 116.1,184.7 169.7,175.8 223.3,57 277,176.8 330.6,176.8 384.3,122.5 437.9,176.8 491.6,176.8 545.2,218.4 598.8,122.5 652.5,184.7 706.1,135.1 "
+                />
+                <rect class="chart-filter" fill="url(#lgrad)" x="0" y="0" width="300%" height="100%"></rect>
+            </svg>
+        `
+
+    ;
+
+    return {
+       bar,
+       donut,
+       line
+    };
+});
```

Comment:
Not really important, and I don't really know why, but it seems that we always need to add a space before the end of the files.